README.md

```diff
@@ -72,12 +72,16 @@ https://llvm.org/docs/GettingStarted.html.
 We highly recommend using the same compiler to compile both llvm and f18.
 
 The f18 CMakeList.txt file uses
-the variable `LLVM_DIR` to find the installed components.
+the variable `LLVM_DIR` to find the installed LLVM components
+and
+the variable `MLIR_DIR` to find the installed MLIR components.
 
-To get the correct LLVM libraries included in your f18 build,
-define LLVM_DIR on the cmake command line.
+To get the correct LLVM and MLIR libraries included in your f18 build,
+define LLVM_DIR and MLIR_DIR on the cmake command line.
 ```
-LLVM=<LLVM_BUILD_DIR>/lib/cmake/llvm cmake -DLLVM_DIR=$LLVM ...
+LLVM=<LLVM_BUILD_DIR>/lib/cmake/llvm \
+MLIR=<LLVM_BUILD_DIR>/lib/cmake/mlir \
+cmake -DLLVM_DIR=$LLVM -DMLIR_DIR=$MLIR ...
```

Comment:
Above 3 lines can be replaced by,
```
cmake -DCMAKE_PREFIX_PATH=<LLVM_BUILD_DIR>
```

Reply:
Good catch @isuruf.  I'll update the instructions in a subsequent PR, along with .drone.io files.  The reason for delay is because I'd like to check this method in my own build farm.

Reply:
Sure. See https://cmake.org/cmake/help/latest/command/find_package.html#search-procedure for more info on why it works. You can also use `cmake --debug-find` with 3.17.0 and `cmake` will tell you which paths that it tried.